std/bitmanip.d

```diff
@@ -247,12 +248,39 @@ private template createTaggedReference(T, ulong a, string name, Ts...)
 }
 
 /**
-Allows creating bit fields inside $(D_PARAM struct)s and $(D_PARAM
-class)es.
-
-The type of a bit field can be any integral type or enumerated
-type. The most efficient type to store in bitfields is $(D_PARAM
-bool), followed by unsigned types, followed by signed types.
+Allows creating `bitfields` inside `structs`, `classes` and `unions`.
+
+A `bitfield` consists of one or more entries with a fixed number of
+bits reserved for each of the entried. The types of the enties can be
+`bool`s, integral types or enumerated types, arbitrarily mixed. The
+most efficient type to store in `bitfields` is `bool`, followed by
+unsigned types, followed by signed types.
+
+Integral and Enumerated types are accompanied by two `enums`: `min`
+and `max`. The numbers that can be represented by the entry of the
+`bitfield` are in the interval [`min`, `max`]. For a given bitfield
+entry `x`, these limits are represented by the name of the entry
+followed by an underscore and the limit name, e.g. `x_min` or
+`x_max`.
```

Comment:
I see, so there are generated behind the scenes and the user can access them by using _min/_max.
I think that this paragraph should be rephrased to make this clear.

Something like: "Each non-bool entry of the bitfield will be represented by an arbitrary number of bits. The minimum and the maxim numbers that represent this domain can be queried by using the name of the variable followed by _min or _max"

Reply:
Actually, the number of bits is not arbitrary, it is user-specified.